mmv1/third_party/terraform/tests/data_source_google_iam_policy_test.go

```diff
@@ -0,0 +1,66 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccComputeImageIamPolicy_googleIamPolicyDataSource(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": randString(t, 10),
+		"role":          "roles/compute.imageUser",
+		"member":        "user:admin@hashicorptest.com",
+		"title":         "expires_after_2019_12_31",
+		"expression":    "request.time < timestamp('2020-01-01T00:00:00Z')",
+	}
+
+	// Note: `<` is changed to `\u003c` in the expression
+	expectedComputedPolicyData := Nprintf(`{"bindings":[{"members":["%{member}"],"role":"%{role}"},{"condition":{"expression":"request.time \u003c timestamp('2020-01-01T00:00:00Z')","title":"%{title}"},"members":["%{member}"],"role":"%{role}"}]}`, context)
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeImageIamPolicy_googleIamPolicyDataSource(context),
+				Check:  resource.TestCheckResourceAttr("data.google_iam_policy.foo", "policy_data", expectedComputedPolicyData),
```

Comment:
just want to double-check that `data` is correct here and not `resource`? I would've thought `resource` was necessary to see the end state of the resource after apply.